build/e2e-image/e2e.sh

```diff
@@ -13,9 +13,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+FEATURES=$1
+echo $FEATURES
 set -e
 echo "installing current release"
-DOCKER_RUN= make install
+DOCKER_RUN= make install FEATURE_GATES='"'$FEATURES'"'
```

Comment:
This `'"'` is needed to prevent `&` from being used as new command:
```
helm upgrade --install --wait --namespace=agones-system\                             
--set agones.featureGates=123&saf \ 
agones /go/src/agones.dev/agones/install/helm/agones/     
bin/sh: saf: command not found   
```

Reply:
In theory you could also use something like "none=false", or something else nonsensical. Just an idea.